### DISCUSSION TITLE
Jenga Example with OrbitControls

### DISCUSSION THREAD

**Discussion by Qberon** [enable3d/enable3d#271] [Help / Questions] [answered]
[You have a very nice example about drag and drop object ](https://enable3d.io/examples/jenga-game.html)

```
// since the scene has a rotation of -Math.PI / 4,
// we adjust the movement by -Math.PI / 4
const v3 = new THREE.Vector3(movementX, 0, movementZ)
v3.applyAxisAngle(new THREE.Vector3(0, 1, 0), -Math.PI / 4)
```
How we can change hardcoded -Math.PI / 4 value to dynamic angle depends on OrbitControls?

**Comment by Qberon** [comment, marked as answer]
Figured out.
Just add OrbitControls and replace -Math.PI / 4 with **orbitControls.getAzimuthalAngle()**

Thanks for a nice framework

**Reply by yandeu** [reply]
> Thanks for a nice framework

Thanks :)
